log/log_test.go

```diff
@@ -103,125 +108,47 @@ func (s *RelayTestSuite) TearDownSuite(t *C) {
 	expect = []proto.LogEntry{
+		{Ts: test.Ts, Level: proto.LOG_INFO, Service: "test", Msg: "info"},
 		{Ts: test.Ts, Level: proto.LOG_WARNING, Service: "test", Msg: "warning"},
 		{Ts: test.Ts, Level: proto.LOG_ERROR, Service: "test", Msg: "error"},
 		{Ts: test.Ts, Level: proto.LOG_CRITICAL, Service: "test", Msg: "fatal"},
 	}
-	t.Check(got, DeepEquals, expect)
-}
-
-func (s *RelayTestSuite) TestLogFile(t *C) {
-	/**
-	 * This test is going to be a real pain in the ass because it writes/reads
-	 * disk and the disk can be surprisingly slow on a test box.  On top of that,
-	 * there's concurrency so we also have to wait for the CPU to run goroutines.
-	 */
-
-	r := s.relay
-	l := s.logger
-
-	// Online log should work without file log.
-	l.Warn("It's a trap!")
-	got := test.WaitLog(s.recvChan, 1)
-	expect := []proto.LogEntry{
-		{Ts: test.Ts, Level: proto.LOG_WARNING, Service: "test", Msg: "It's a trap!"},
-	}
-	t.Check(got, DeepEquals, expect)
-
-	log, err := ioutil.ReadFile(s.logFile)
-	if !os.IsNotExist(err) {
-		t.Error("We haven't enabled the log file yet, so it shouldn't exist yet")
-	}
-
-	// Enable the log file.
-	r.LogFileChan() <- s.logFile
-
-	// Online log should work with the file log.
-	l.Warn("It's another trap!")
-	got = test.WaitLog(s.recvChan, 1)
-	expect = []proto.LogEntry{
-		{Ts: test.Ts, Level: proto.LOG_WARNING, Service: "test", Msg: "It's another trap!"},
-	}
-	t.Check(got, DeepEquals, expect)
-
-	// Log file should exist.
-	size, _ := test.FileSize(s.logFile)
-	test.WaitFileSize(s.logFile, size)
-	log, err = ioutil.ReadFile(s.logFile)
-	if err != nil {
-		t.Fatalf("Log file should exist: %s", err)
-	}
-
-	if !strings.Contains(string(log), "It's another trap!") {
-		t.Error("Log file contains entry after being enabled, got\n", string(log))
-	}
-	if strings.Contains(string(log), "It's a trap!") {
-		t.Error("Log file does not contain entry before being enabled, got\n", string(log))
-	}
-
-	l.Debug("Hello")
-	if strings.Contains(string(log), "Hello") {
-		t.Error("Log file should respect the log level")
-	}
-
-	// We should be able to change the log file.
-	newLogFile := s.logFile + "-new"
-	defer func() { os.Remove(newLogFile) }()
-	r.LogFileChan() <- newLogFile
-	l.Warn("Foo")
-
-	size, _ = test.FileSize(newLogFile)
-	test.WaitFileSize(newLogFile, size)
-	log, err = ioutil.ReadFile(newLogFile)
-	if err != nil {
-		t.Errorf("New log file should exist: %s", err)
-		return
-	}
-
-	if !strings.Contains(string(log), "Foo") {
-		t.Error("New log file contains only the new log entry, got\n", string(log))
-	}
-	if strings.Contains(string(log), "It's another trap!") {
-		t.Error("New log file should contain only the new log entry")
-	}
-
-	log, err = ioutil.ReadFile(s.logFile)
-	if err != nil {
-		t.Errorf("Old log file should still exist: %s", err)
-		return
-	}
-
-	if strings.Contains(string(log), "Foo") {
-		t.Error("Old log file should not contain the new log entry")
-	}
+	assert.Equal(t, expect, got)
 }
```

Comment:
It looks like we don't log anymore to file, only to `stdout` and `stdin` so I've removed this test as there is functionality it could test